Compilation error on macOS

```
/Users/rex/xxHash on branch dev rex@MacBook-Pro-2018% make
cc -O3     -c -o xxhash.o xxhash.c
ar rcs libxxhash.a xxhash.o
cc -O3   -fPIC  xxhash.c -shared -install_name /usr/local/lib/libxxhash.0.dylib -compatibility_version 0 -current_version 0.8.0 -o libxxhash.0.8.0.dylib
ln -sf libxxhash.0.8.0.dylib libxxhash.0.dylib
ln -sf libxxhash.0.8.0.dylib libxxhash.dylib
cc -O3     -c -o xxhsum.o xxhsum.c
cc -O3     -c -o cli/xsum_os_specific.o cli/xsum_os_specific.c
cc -O3     -c -o cli/xsum_output.o cli/xsum_output.c
cc -O3     -c -o cli/xsum_sanity_check.o cli/xsum_sanity_check.c
In file included from cli/xsum_sanity_check.c:27:
cli/xsum_sanity_check.h:51:52: error: unknown type name 'size_t'
XSUM_API void XSUM_fillTestBuffer(XSUM_U8* buffer, size_t len);
                                                   ^
1 error generated.
make: *** [cli/xsum_sanity_check.o] Error 1
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -42,6 +42,14 @@ matrix:
         - CPPFLAGS=-DXXH_REROLL=1 make check   # reroll code path (#240)
         - make -C tests/bench
 
+    - name: macOS General Test
+      os: osx
+      compiler: clang
+      script:
+        - make   # test library build
+        - make clean
+        - make test MOREFLAGS='-Werror' | tee # test scenario where `stdout` is not the console
+
     - name: ARM compilation and consistency checks (Qemu)
       dist: xenial
       arch: amd64
@@ -111,7 +119,7 @@ matrix:
         - CPPFLAGS=-DXXH_VECTOR=XXH_VSX CFLAGS="-O3 -maltivec -mvsx -mpower8-vector -mcpu=power8" LDFLAGS="-static" make check
         # altivec.h redefinition issue #426
         - make clean
-        - CPPFLAGS=-DXXH_VECTOR=XXH_VSX CFLAGS="-maltivec -mvsx -mcpu=power8 -mpower8-vector" make -C tests test_ppc_redefine 
+        - CPPFLAGS=-DXXH_VECTOR=XXH_VSX CFLAGS="-maltivec -mvsx -mcpu=power8 -mpower8-vector" make -C tests test_ppc_redefine
 
     - name: IBM s390x compilation and consistency checks
       dist: bionic

--- a/.travis.yml
+++ b/.travis.yml
@@ -2,7 +2,7 @@ language: c
 
 # Dump CPU info before start
 before_install:
-  - cat /proc/cpuinfo
+  - cat /proc/cpuinfo || echo /proc/cpuinfo is not present
 
 matrix:
   fast_finish: true

--- a/cli/xsum_sanity_check.h
+++ b/cli/xsum_sanity_check.h
@@ -26,7 +26,10 @@
 #ifndef XSUM_SANITY_CHECK_H
 #define XSUM_SANITY_CHECK_H
 
-#include "xsum_config.h"
+#include "xsum_config.h"  /* XSUM_API, XSUM_U8 */
+
+#include <stddef.h>   /* size_t */
+
 
 #ifdef __cplusplus
 extern "C" {

